src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -89,12 +89,12 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             InitializeComponent()
 
             ' Support sorting.
-            _comparer = New ListViewComparer With {
-                .SortColumn = 0,
-                .Sorting = SortOrder.Ascending
-            }
-            listViewExtensions.ListViewItemSorter = _comparer
-            listViewExtensions.Sorting = SortOrder.Ascending
+            '_comparer = New ListViewComparer With {
```

Comment:
Don't leave this code commented out - it should either be replaced/updated, or just deleted if for some reason we don't need it anymore.

Reply:
Done